src/content/blog/2024-07-05-whats-coming-next-for-eslint.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: What's coming next for ESLint
+teaser: "The new configuration system is just the beginning of significant changes coming to ESLint."
+tags:
+  - Languages
+  - Tools
+  - Architecture
+authors:
+  - nzakas
+categories:
+  - Announcements
+---
+
+When we [released ESLint v9.0.0](https://eslint.org/blog/2024/04/eslint-v9.0.0-released/) in April, it was the first major release in over 18 months and formally introduced the [new](https://eslint.org/blog/2022/08/new-config-system-part-1/) [configuration](https://eslint.org/blog/2022/08/new-config-system-part-2/) [system](https://eslint.org/blog/2022/08/new-config-system-part-3/). ESLint v9.0.0 also made some [rule API changes](https://eslint.org/blog/2023/09/preparing-custom-rules-eslint-v9/) to prepare the core for what's coming next. After the release, we spent a lot of time [creating compatibility utilities](https://eslint.org/blog/2024/05/eslint-compatibility-utilities/), a [configuration migration tool](https://eslint.org/blog/2024/05/eslint-configuration-migrator/), and a [rule API transform utility](https://github.com/eslint/eslint-transforms) to help the ecosystem move to ESLint v9.0.0. All of that work was necessary for us to move on to the next significant changes coming to ESLint.
```

Comment:
v8.0.0 was released on October 9, 2021 so this is more like ~30 months.

Reply:
Also, are three links on three words in a row intentional? Looks a bit unusual.

```md
[new](https://eslint.org/blog/2022/08/new-config-system-part-1/) [configuration](https://eslint.org/blog/2022/08/new-config-system-part-2/) [system](https://eslint.org/blog/2022/08/new-config-system-part-3/)
```

![image](https://github.com/eslint/eslint.org/assets/44349756/35fdea2e-c61f-47a8-8600-f8ad3c3675cb)

Reply:
Yes, that's intentional.